src/PolarSSL++/BlockingSslClientSocket.cpp

```diff
@@ -189,7 +189,7 @@ bool cBlockingSslClientSocket::Send(const void * a_Data, size_t a_NumBytes)
 	size_t NumBytes = a_NumBytes;
 	for (;;)
 	{
-		int res = m_Ssl.WritePlain(a_Data, a_NumBytes);
+		int res = m_Ssl.WritePlain(Data, a_NumBytes);
```

Comment:
Fixed use of wrong variable. The method `Send` is not used, but better fix that.